locale/en-US/tbSync.strings

```diff
@@ -4,7 +4,7 @@ info.sync=Syncing
 
 button.syncthis=Synchronize this account
 button.tryagain=Try again to connect to server
-button.enableAndConnect=Enable account & try to connect to server
+button.enableAndConnect=Enable account and try to connect to server
```

Comment:
I want to keep the &

Reply:
The text 'and' is better; however, reverted change to your preferred ampersand.